modules/heat_conduction/doc/content/source/postprocessors/ExposedSideAverageValue.md

```diff
@@ -0,0 +1,13 @@
+# ExposedSideAverageValue
+
+!syntax description /Postprocessors/ExposedSideAverageValue
+
+This is used to compute the average of a variable over only the portion of a specified surface that is exposed, using the [SelfShadowSideUserObject](SelfShadowSideUserObject.md) to determine which portions of the surface are exposed and which are shadowed. This computation is identical to that performed in [SideAverageUserObject](SideAverageUserObject.md), except that only portions of the surface that are exposed (i.e., not shadowed) are included in the calculation.
+
+!listing directional_flux_bc/2d.i block=Postprocessors/ave_v_exposed
```

Comment:
This is not useful unless you add text to say what s being done.
The !syntax inputs /Postprocessors/ExposedSideAverageValue lists this input already

Reply:
Removed